samples/v1/test/samples.manifest.yaml

```diff
@@ -0,0 +1,38 @@
+type: manifest/samples
+schema_version: 3
+base: &common
+  env: 'nodejs'
+  bin: 'node'
+  chdir: '{@manifest_dir}/../..'
+  basepath: '.'
+samples:
+- <<: *common
```

Comment:
Could you provide an example of how these manifest files will be used, I heard a rumor there was a new python testing harness?

How will we run individual sample tests, the whole suite, etc?

One concern I have about a Python testing harness is that it would side-step some of our Node.js tooling like test coverage, I would like to make sure we configure a Python test harness such that:

1. it can still be run via `npm run samples-test`.
2. we're still able to instrument for coverage, this will be a great indicator of whether we've written comprehensive samples for the library.

When we're ultimately working on wiring up tests to CI/CD, I'd love to be looped in.